mobcat_shared/MobCAT/MVVM/AsyncCommand.cs

```diff
@@ -38,8 +38,14 @@ public AsyncCommand(Func<Task> action, Func<bool> canExecute) : this(o => action
         /// <inheritdoc />
         public bool CanExecute(object parameter)
         {
+            if (_canExecute == null)
```

Comment:
Fix for app crash when _canExecute is null

Reply:
omg good catch!

Reply:
Thanks! 😄